resources/dtr_bundle_patient_pat015.json

```
@@ -0,0 +1,1332 @@
+        "valueQuantity": {
+          "value": 95,
+          "unit": "%",
+          "system": "http://unitsofmeasure.org",
+          "code": "%"
+        },
+        "encounter": {
+          "reference": "Encounter/pat015-rad-encounter"
+        },
+        "performer": [
+          {
+            "reference": "Practitioner/pra-dmorgan",
+            "display": "Dexter Morgan",
+            "type": "Practitioner"
+          },
+          {
+            "reference": "Organization/org1234",
+            "display": "Gulf Coast Lab",
+            "type": "Organization"
+          }
+        ]
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Observation/obs015-o2sat-resting"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Observation",
+        "id": "obs015-o2sat-treatment",
+        "status": "final",
+        "category": [
+          {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+                "code": "laboratory",
+                "display": "Laboratory"
+              }
+            ],
+            "text": "Laboratory"
+          }
+        ],
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "59409-3",
+              "display": "Oxygen saturation in Arterial blood by Pulse oximetry --during treatment"
+            }
+          ]
+        },
+        "subject": {
+          "reference": "Patient/pat015"
+        },
+        "issued": "2020-07-01T15:30:10+01:00",
+        "valueQuantity": {
+          "value": 97,
+          "unit": "%",
+          "system": "http://unitsofmeasure.org",
+          "code": "%"
+        },
+        "encounter": {
+          "reference": "Encounter/pat015-rad-encounter"
+        },
+        "performer": [
+          {
+            "reference": "Practitioner/pra-dmorgan",
+            "display": "Dexter Morgan",
+            "type": "Practitioner"
+          },
+          {
+            "reference": "Organization/org1234",
+            "display": "Gulf Coast Lab",
+            "type": "Organization"
+          }
+        ]
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Observation/obs015-o2sat-treatment"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Observation",
+        "id": "obs-pat015-pao2",
+        "category": [
+          {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+                "code": "laboratory",
+                "display": "Laboratory"
+              }
+            ],
+            "text": "Laboratory"
+          }
+        ],
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "2703-7",
+              "display": "Oxygen (BldA) [Partial pressure]"
```

```suggestion
              "display": "Oxygen [Partial pressure] in Arterial blood"
```
Code display doesn't match, this can be a warning but is an error by default